### PR TITLE
Add new st2bundle dependencies

### PR DESCRIPTION
**Pull request by armab**
Reflects changes: https://github.com/StackStorm/st2-packages/pull/101

### CODE REVIEW

File: stackstorm/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM debian:wheezy
 ARG ST2_VERSION
 LABEL com.stackstorm.version="${ST2_VERSION}"
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2
+RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev
```
Comment (dennybaa) [inline stackstorm/Dockerfile]: what actual version of libffi we need. I guess `libffi5` and `libffi-dev` are inconsistent...

Comment (armab) [inline stackstorm/Dockerfile]: The build is passed with this change, Docker images are in Hub:
Failed build: https://circleci.com/gh/StackStorm/st2/470
Fixed build: https://circleci.com/gh/StackStorm/st2/475

**Comment by dennybaa** [comment]
@armab  I'm talking about consistency when libffi5 and libffi-dev. Not talking about builds. Of course they passing. Just for info it might be that libffi-dev installs version 6, and what for then you need version 5. Idea is about this.

**Comment by armab** [comment]
I see, so it's the question "in general".

`libffi5` is linked automatically when we build that wheezy package under the wheezy environment.

Bintray shows entire metadata for uploaded artifact, so this is how resulting dependcies look like.
See `Depends` here: https://bintray.com/stackstorm/wheezy_staging/st2bundle/view


I have no idea in practice how one is consistent with another, but `libffi5` is what we get during package building.

**Comment by armab** [comment]
Found it: 

`libffi-dev` + `libffi5` looks perfectly nice together:
https://packages.debian.org/wheezy/libffi-dev
sine `libffi-dev` depends on `libffi5`.

**Comment by dennybaa** [comment]
check also trusty please?

**Comment by dennybaa** [comment]
since my suggestion to use
`libffi libffi-dev` it's more consistent, and doesn't install useless pieces when used on different distros.

**Comment by armab** [comment]
Keep in mind that this is Dockerfile for `wheezy`. We can't get rid of `libffi5` for `wheezy` I think and should hardcode it here with current `st2bundle` installation way.

----

For `trusty` `libffi-dev` depends on `libffi6`: https://launchpad.net/ubuntu/trusty/amd64/libffi-dev/3.1~rc1+r3.0.13-12

Which is again linked by default in `st2bundle` built under `trusty` environment: https://bintray.com/stackstorm/trusty_staging/st2bundle/view

So far looks good :+1:

**Comment by dennybaa** [comment]
noop just consistency and additional package won't be fetched if you just use `libffi libffi-dev`

**Pull request merged**